In [1]:
!pip install latex

  Stored in directory: C:\Users\Renate\AppData\Local\pip\Cache\wheels\a5\00\74\5aed853dec3fbc45e9c9cf4949f664f1bd8b6f3e6def432019
  Stored in directory: C:\Users\Renate\AppData\Local\pip\Cache\wheels\7b\f2\40\6806964a69dfe3e46eff51b06a9a036af80b933479536fa295
  Stored in directory: C:\Users\Renate\AppData\Local\pip\Cache\wheels\f6\09\e5\5afbc45fb0de471541092c7cd8b48d3483be68b1890955e6b9
  Stored in directory: C:\Users\Renate\AppData\Local\pip\Cache\wheels\05\ef\eb\27a815601fa45bbb0301dae456f0853700502233b801fe4bb1
Successfully built latex tempdir data shutilwhich


In [2]:
import latex

$$\alpha = 0.05$$

$$\bar y \sim N\left({\mu_y}, \frac{\sigma_y}{\sqrt n}\right)$$

This is the Central Limit Theorem: $\bar y \sim N\left({\mu_y}, \frac{\sigma_y}{\sqrt n}\right)$

$$\sum y \sim N\left({n\mu_y}, {\sqrt n}\cdot{\sigma_y}\right)$$

## **Exercise 1**

Let's start with an exercise (#4.85 in the book from Ott&Longnecker, p.255): 

The baggage limit for an airplane is set at 100 pounds per passenger. 
Thus, for an airplane with 200 passenger seats, there would be a limit of 
20.000 pounds. The weight of the baggage of an individual passenger is 
a random variable with a mean of 95 pounds and a standard deviation 
of 35 pounds. If all 200 seats are sold for a particular flight, what is the 
probability that the total weight of the passengers' baggage will exceed
the 20.000 pound limit?

-------------------------------------------------------------------------------------------------------------

## **Answer E1**

y = the weight of the baggage of an individual passenger

w = the sum of all y-values $\sum_i^{200}  y_i = y_1 + \cdots + y_{200}$

w $\sim_{approx} N\left({\mu_w},{\sigma_w}\right)$ = $N({95x200}, {\sqrt{200}\cdot{35}})$ = $N({19000}, {494.97})$

P = probability
$P(w > 20000)$ = $1 - P(w > 20000)$ = $1 - P(w <= 20000)$

## Exercise 2

Next, we will compute the CLT of the example on page 198 in the book from Ott & Longnecker.

A person visits her doctor with concerns about her blood pressure. If the systolic blood pressure exceeds 150, the patient is considered to have high blood pressure and medication may be prescribed. A patient's blood pressure readings often have a considerable variation during a given day. Suppose a patient's systolic blood pressure readings during a given day have a normal distribution with a mean of $\mu$ = 160 mm mercury and a standard deviation $\sigma$ = 20 mm

### a. question:
What is the prbability that a single blood pressure measurement will fail to dectect that the patient has high blood pressure? 

### a. answer:

$P$(measurement fails to detect high pressure) = 
$P(y \leqq 150) = $
$P(z \leqq \frac{(150-160)}{20}) = $
$p(Z \leqq -0.5) = .3805$

Thus, the chance of failing to detect the that patient has a high blood pressure if only a single measurement is taken is over 30%. (too high)

### b. question: 
If five blood pressure measuments are taken at various times during the day, what is the probability that the average of the five measurements will be less than 150 and hence fail to indicate that the patient has high blood pressure? 

### b. answer:

let $ \bar y$ be the average blood pressure of the five measurements. Then $\bar y$ has a normal distribution with $\mu $ = 160 and $\sigma = 20$ /$ \sqrt {5} $ = 8.944

In [1]:
import math
x = math.sqrt(5)
print(x)

2.23606797749979


In [2]:
i = 20 
print(i/x)

8.94427190999916


$P(\bar y \leqq 150) = $
$P(z \leqq \frac{(150-160)}{8.944}) = $
$P(z \leqq -1.12) = .1314$

In [4]:
#Same calculation with scipy
from scipy.stats import norm
norm.cdf(-1.12)

0.1313568810427307

Therefore, by using the average of five measurements, the chance of failing to detect the patient has high blood pressure has been reduced from over 30% to about 13%. 

### c. question:
How many measurements would be required in a given day so that there is at most 1% probability of failing to detect that the patient has high blood pressure? 

### c. answer:
We need to determine the sample size $n$ such that $P(\bar y < 150)\leqq 0.01$

first let's calculate $z$

In [3]:
from scipy.stats import norm
norm.ppf(0.01)

-2.3263478740408408

Ok, so now we know that the solution is -2.326

Therefore $\frac{150 - 160}{20/\sqrt n} = -2.326$

Solving for $n$ yiels $\sqrt n = \frac{(-2.326)(20)}{-10} $
$n = 21.64$

#### Conclusion: 
It would take at least 22 measurements in order to achieve the goal of at most a 1% chance of failing to detect high blood pressure.